src/includes/classes/MenuPageOptions.php

```diff
@@ -697,6 +697,28 @@ public function __construct()
 
         echo '<div class="plugin-menu-page-panel">'."\n";
 
+        if (is_multisite() && $this->plugin->applyWpFilters(GLOBAL_NS.'_exclude_hosts_option_enable', is_multisite())) {
```

Comment:
This line should be:

```php
if ($this->plugin->applyWpFilters(GLOBAL_NS.'_exclude_hosts_option_enable', is_multisite())) {
```

Meaning, the filter is `true` on a network by default, and the panel is shown. On a standard install, `is_multisite()` is false and the filter will default to being off. In either case, a developer can override by attaching to and altering the filtered value used in the conditional.